godtools/App/Share/Domain/UseCases/GetBannerImageUseCase/GetBannerImageUseCase.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  GetBannerImageUseCase.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 8/8/22.
+//  Copyright © 2022 Cru. All rights reserved.
+//
+
+import Foundation
+import Combine
+import SwiftUI
+
+class GetBannerImageUseCase {
+    
+    private let attachmentsRepository: AttachmentsRepository
+    
+    init(attachmentsRepository: AttachmentsRepository) {
+        
+        self.attachmentsRepository = attachmentsRepository
+    }
+    
+    func getBannerImagePublisher(for attachmentId: String) -> AnyPublisher<Image?, Never> {
+        
+        return attachmentsRepository.getAttachmentImage(id: attachmentId)
+    }
+    
+    func getBannerImage(for attachmentId: String) -> Image? {
```

Comment:
Hey @rachaelblue this PR looks good.  Eventually I would like this method to go away since this method will only work on cached banner images and is only useful for SwiftUI Previews.  Instead we can have a Development Assets Catalog for SwiftUI Previews.  This work doesn't isn't a showstopper for the next TestFlight release and can be done as part of the refactoring in this task here. -> https://jira.cru.org/browse/GT-1755